modules_core/tabs.js

```diff
@@ -191,6 +191,17 @@ exports.create = function (api) {
     })
 
     if (process.versions.electron) {
+      var {webFrame} = require('electron')
+      
+      window.addEventListener('mousewheel', function (ev) {
+        var {ctrlKey, deltaY} = ev
+        if(ctrlKey){
+          var direction = (deltaY / Math.abs(deltaY))
+          var currentZoom = webFrame.getZoomLevel()
+          webFrame.setZoomLevel(currentZoom - direction)
+        }
```

Comment:
this looks great. I'll smoke test then merge.

I'm gonna change some of this to be styled a little more consisten with what I've been writing with es6.